homework01/vector.h

```diff
@@ -1,5 +1,35 @@
 
 class Vector
 {
-	// TODO
+ public:
+  Vector(double x = 0., double y = 0.)
+    {
+      this->x = x;
+      this->y = y;
```

Comment:
You can use `this->`, there is nothing wrong with it, although it's not very popular to do that in C++: it makes the code extremely verbose for little added value. Our convention at Optiver is to use `mAttribute`.